ERROR_MESSAGES.md

```diff
@@ -35,7 +35,7 @@ Any text in the following format `(Example)` are considered variables to be fill
 * 34 : CT - Must contain a practice site address for CPC+ conversions
 * 35 : CT - One and only one Alternative Payment Model (APM) Entity Identifier should be specified
 * 36 : CT - CPC+ submissions must contain one Measure section
-* 37 : CT - Must contain correct number of performance rate(s). Correct Number is `(Expected value)`
+* 37 : CT - CPC+ submissions must contain correct number of performance rate(s). Correct Number is `(Expected value)` for measure `(Given measure id)`
```

Comment:
Do we want to include the extra ``for measure `(Given measure id)` `` when I just merged PR #711 which adds a reference to the measure in the JSON returned to the user?

Reply:
I don't see the harm so long as the value is consistent. Will give #711 a look.